app/Module.scala

```diff
@@ -1,16 +1,49 @@
+import java.io.{File, FileInputStream, InputStreamReader}
+import java.util
 import java.util.function.Function
 
 import actors._
 import akka.actor.ActorRef
+import com.google.api.client.auth.oauth2.{Credential, StoredCredential, TokenResponse}
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver
+import com.google.api.client.googleapis.auth.oauth2.{GoogleAuthorizationCodeFlow, GoogleClientSecrets}
+import com.google.api.client.http.javanet.NetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.client.util.store.{DataStore, FileDataStoreFactory}
+import com.google.api.services.youtube.YouTube
+import com.google.common.collect.Lists
 import com.google.inject.AbstractModule
 import com.google.inject.name.Names
 import com.google.inject.util.Providers
 import controllers.{DefaultKnolxControllerComponents, KnolxControllerComponents}
 import net.codingwell.scalaguice.ScalaModule
 import play.api.libs.concurrent.AkkaGuiceSupport
+import play.api.{Configuration, Environment}
 import play.libs.Akka
 
-class Module extends AbstractModule with ScalaModule with AkkaGuiceSupport {
+class Module(environment: Environment,
```

Comment:
environment and configuration does not seem to be used anywhere.
Please self review the pull request and let me know once it is done.